### ISSUE TITLE
Render Eddystone TLM beacons

### ISSUE THREAD

**Issue by wilsonpage** [mozilla-magnet/magnet-client#8]
> Eddystone beacons may transmit data about their own operation to clients. This data is called telemetry and is useful for monitoring the health and operation of a fleet of beacons [...](https://github.com/google/eddystone/tree/master/eddystone-tlm)

Not all beacons emit URLs. It might be cool to render temperature (or other advertised data) when a relevant beacon is discovered.

**Comment by wilsonpage** [comment]
We can simulate TLM broadcasts using the [node-eddystone-beacon](https://github.com/don/node-eddystone-beacon) library.

**Comment by wilsonpage** [comment]
@arcturus do you have those Estimote beacons still?

**Comment by arcturus** [comment]
Yes I have them.

But remember they use a completely different protocol.

**Comment by wilsonpage** [comment]
I thought they were Eddystone?

**Comment by wilsonpage** [comment]
http://developer.estimote.com/eddystone/

**Comment by arcturus** [comment]
Right, but we got the estimote stickers, that support just iBeacon and their own protocol (nearable or something like that):

http://developer.estimote.com/nearables/stickers-vs-beacons/

**Comment by wilsonpage** [comment]
Balls. Still worth supporting them?

**Comment by arcturus** [comment]
Not so sure, I think it will distract us.

As a hack, perhaps, but it's something quite specific.

Anyway the idea of supporting Eddystone TLM is good, we can have rpi3 advertising a beacon like that.

**Comment by wilsonpage** [comment]
From the Eddystone spec it seems TLM supports transmitting beacon battery life or temperature. Any idea if we can use TLM type to transfer arbitrary data too?

**Comment by wilsonpage** [comment]
> Not so sure, I think it will distract us.

Agreed :)
